src/Surfnet/StepupMiddleware/ApiBundle/Tests/Authorization/Service/CommandAuthorizationServiceTest.php

```diff
@@ -314,7 +315,10 @@ public function an_identity_should_not_be_able_to_execute_someone_elses_selfserv
                 ->andReturn($raCredentials);
 
             // These commands should always be allowed to be executed
-            if ($command instanceof CreateIdentityCommand || $command instanceof UpdateIdentityCommand) {
+            if ($command instanceof CreateIdentityCommand ||
```

Comment:
Maybe also extract this If condition to a seperate method?  Function alwaysExecutableCommand or something like that.  That'd allow to get rid of the comment.

Reply:
Given this is test code, I feel comfortable leaving this as is.